drivers/interrupt_controller/rv32m1_intmux.c

```diff
@@ -194,4 +194,4 @@ static int rv32m1_intmux_init(struct device *dev)
 DEVICE_AND_API_INIT(intmux, DT_OPENISA_RV32M1_INTMUX_INTMUX_LABEL,
 		    &rv32m1_intmux_init, NULL,
 		    &rv32m1_intmux_cfg, PRE_KERNEL_1,
-		    CONFIG_KERNEL_INIT_PRIORITY_DEFAULT, &rv32m1_intmux_apis);
+		    CONFIG_RV32M1_INTMUX_INIT_PRIORITY, &rv32m1_intmux_apis);
```

Comment:
Current setting CONFIG_KERNEL_INIT_PRIORITY_DEFAULT is 40, but CONFIG_RV32M1_INTMUX_INIT_PRIORITY is 60.  Is that the actual fix?

Reply:
@mike-scott The real fix is to honour the init priority set by the user in Kconfig. The default has changes, yes, but I don't see any problem arising from that change.

Reply:
Would changing the default to 40 break your I2C enablement work? If not, maybe including that as a change in this PR would be a good way to minimize risk from this patch.

Reply:
@mbolivar Thanks, good idea. I have included that change.